emails/models.py

```diff
@@ -833,6 +835,7 @@ def save(
                 update_fields = {"block_list_emails"}.union(update_fields)
         if (not user_profile.server_storage) and self.description:
             self.description = ""
+            self.used_on = ""
             if update_fields:
                 update_fields = {"description"}.union(update_fields)
```

Comment:
need to update `update_fields` for `used_on`:

```suggestion
        if (not user_profile.server_storage) and (self.description or self.used_on):
            self.description = ""
            self.used_on = ""
            if update_fields:
                update_fields = {"description", "used_on"}.union(update_fields)
```